### DISCUSSION TITLE
Resizing Main Window or SidePanel

### DISCUSSION THREAD

**Discussion by mjehrhart** [emilk/egui#1522] [Q&A]
I think the answer is No but I would like to confirm.  Is it possible to resize the main widow - I would like it to shrink and expand from left to right. FYI, I use a CentralPanel and SidePanel in the main window. Or if the window cannot be resized, can the SidePanel be resized?

In the image, I would like the app to shrink to the size of the circles and then expand later.
<img width="945" alt="Screen Shot 2022-04-20 at 8 33 25 PM" src="https://user-images.githubusercontent.com/97703291/164370613-4aa254de-c556-429e-91fa-3faf7dd092c5.png">

Thanks,
mje

**Comment by emilk** [comment]
https://docs.rs/eframe/latest/eframe/struct.NativeOptions.html#structfield.resizable
https://docs.rs/epi/0.17.0/epi/struct.Frame.html#method.set_window_size
